clients/python/agentic-sandbox-client/k8s_agent_sandbox/async_k8s_helper.py

```
@@ -37,6 +39,26 @@
+        return max(float(raw_retry), 0.0)
+    except (ValueError, TypeError):
+        pass
+    try:
+        from email.utils import parsedate_to_datetime
+        from datetime import datetime, timezone
+        retry_at = parsedate_to_datetime(raw_retry)
+        if retry_at.tzinfo is None:
+            retry_at = retry_at.replace(tzinfo=timezone.utc)
+        delay = (retry_at - datetime.now(timezone.utc)).total_seconds()
+        return max(delay, 0.0)
+    except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)